@here/harp-mapview/lib/ColorCache.ts

```diff
@@ -30,7 +30,15 @@ export class ColorCache {
      * @param colorCode ThreeJS color code or name. You must provide a valid color code or name,
      * as this function does not do any validation.
      */
-    getColor(colorCode: string): THREE.Color {
+    getColor(colorCode: string | number): THREE.Color {
+        if (typeof colorCode === "number") {
```

Comment:
You could use `leftPad` for this https://www.npmjs.com/package/left-pad

;)

Couldn't resist.

Reply:
Wasn't aware of this! Good suggestion!